README.md

```diff
@@ -58,6 +60,53 @@ spec:
             - Resource
 ```
 
+### Artifact Hub
+1. A dedicated folder must be created for each policy.
+2. The folder must be named the same as the policy.
+3. Use dashes for folder name and policy name instead of underscores.
+
+Add an `artifacthub-pkg.yml` metadata file to the folder. See an example metadata file for Kyverno policies below and customize per the comments.
+```yaml
+---
+name: backup-all-volumes # The name of the package (only alphanum, no spaces, dashes allowed) (required)
+version: 1.0.0 # Version of the policy (required)
+displayName: Backup All Volumes  # Display name of the policy (required)
+createdAt: "2023-03-29T00:00:00.000Z" # The date this package was created (RFC3339 layout) (required)
+description: >-
```

Comment:
Please add comment here indicating the value should be taken from the relevant annotation.